template.yaml

```diff
@@ -891,10 +941,61 @@ Resources:
               - ActionCondition:
                   Action: BLOCK
 
+  BackupFunction:
+    Condition: EnableBackup
+    Type: AWS::Serverless::Function
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W89
+            reason: VPC not required.
+          - id: W92
+            reason: ReservedConcurrentExecutions not required.
+    Properties:
+      Runtime: python3.10
+      Handler: app.lambda_handler
+      MemorySize: 128
+      Timeout: 30
+      CodeUri: functions/backup-function
+      Events:
+        CWSchedule:
+          Type: ScheduleV2
+          Properties:
+            ScheduleExpression: 'rate(1 day)'
+      Policies: 
+        S3WritePolicy:
```

Comment:
I prefer to be specific with these types of policies as sometimes doing this gives more permission than is often needed. Also if we are going to add a prefix/folder then we would need to change this anyway?

Reply:
I'll scope it to the hardcoded prefix